src/quo/components/drawers/action_drawers/view.cljs

```diff
@@ -87,6 +97,16 @@
             :accessibility-label :right-icon-for-action}
            [icon/icon right-icon
             {:color (get-icon-color danger? theme)
+             :size  20}]])
+        (when (= state :selected)
+          [rn/view
+           {:style               style/right-icon
+            :accessible          true
```

Comment:
Is there a specific reason for setting the `accessible` prop? As far as I know, we currently use the accessibility label only for testing purposes.